### ISSUE TITLE
--version-number flag

### ISSUE THREAD

**Issue by comp500** [packwiz/packwiz#226]
Flag similar to `--version-id` but resolves version numbers into version IDs, equivalent to passing a URL.

**Comment by MenacingPerson** [comment]
Actually if the mod has the same version number for different releases, then it will have a different url than the

`https://modrinth.com/mod/<slug>/version/<version>`

It will be the version id instead of version.

**Comment by comp500** [comment]
Yeah, hmm... how should packwiz resolve the version number when there are multiple? The URL handling code matches the Modrinth frontend - i.e. the oldest version gets the version number, the others must use the ID, but I think this might be counter-intuitive for a `--version-number` flag.

**Comment by MenacingPerson** [comment]
I think that it should match all the versions with the same number, then pick the one closest to the pack's platform. If there are still multiple, offer a selector and the default one can be the most recent one.
